src/main/java/org/broadinstitute/dsm/util/externalShipper/GBFRequestUtil.java

```diff
@@ -199,65 +201,52 @@ else if (address != null) {
     //  Status is as it sounds, a real-time status of the order progression through the GBF internal process
     // 'RECEIVED', 'PROCESSING', 'SHIPPED', 'DISTRIBUTION', 'FORMS PRINTED', 'CANCELLED', 'NOT FOUND', 'SHIPPED (SIMULATED)'
     // return the actual tube barcode for all tubes but only after the kit is shipped (around 7:00pm)
-    public void orderStatus(ArrayList<KitRequest> kitRequests) throws Exception {
-        if (kitRequests != null && !kitRequests.isEmpty()) {
-            List<String> orderNumbers = new ArrayList<>();
-            Map<String, KitRequest> externalOrdersStatus = new HashMap<>();
-            for (KitRequest kit : kitRequests) {
-                if (!orderNumbers.contains(kit.getExternalOrderNumber()) && StringUtils.isNotBlank(kit.getExternalOrderNumber())) {
-                    orderNumbers.add(kit.getExternalOrderNumber());
-                    externalOrdersStatus.put(kit.getExternalOrderNumber(), kit);
-                }
-            }
-            JSONObject payload = new JSONObject().put("orderNumbers", orderNumbers);
-            logger.info("payload size for order status: " + orderNumbers.size());
-            String sendRequest = DSMServer.getBaseUrl(getExternalShipperName()) + STATUS_ENDPOINT;
-            logger.info("payload: " + payload.toString());
-            Response gbfResponse = executePost(Response.class, sendRequest, payload.toString(), DSMServer.getApiKey(getExternalShipperName()));
-            logger.info(gbfResponse.getXML());
```

Comment:
This might be blowing things up--big log of a potentially very large string of XML.